e2e/LongHaul/device/Program.cs

```diff
@@ -110,6 +110,7 @@ await Task
                     .ConfigureAwait(false);
             }
             catch (OperationCanceledException) { } // user signalled an exit
+            catch (AggregateException ex) when (ex.InnerException is OperationCanceledException) { } // user signaled an exit
```

Comment:
Any particular cases while seeing this exception?

Reply:
When more than 1 task had an exception. Made the same as service client code.